pycon-submission.md

```diff
@@ -12,87 +25,124 @@ Intermediate
 
 # Domain Level
 
-Intermediate
+Introductory
 
 # Description
 
-IPython and Jupyter provide tools for interactive and parallel computing that are widely
+IPython and Jupyter provide tools for interactive computing that are widely
 used in scientific computing, but can benefit any Python developer.
 
-We will show how to use IPython in different ways, as: an interactive shell,
-a graphical console, a network-aware VM in GUIs, a web-based
-notebook with code, graphics and rich HTML, and a high-level framework for
-parallel computing.
+We will show how to use IPython in different ways, as:
+
+- an interactive shell,
+- a graphical console,
+- a network-aware VM in GUIs,
+- a web-based notebook with code, graphics and rich HTML.
 
-# Audience 
+We will demonstrate how to deploy a custom environment
+with Docker that not only contains multiple Python kernels but also a couple
+of other languages.
+
+# Audience
 
 Programmers interested in using Python interactively, especially in data
-analysis environments.
+analysis environments. Prior knowledge of Python is best. Some experience
```

Comment:
Some prior knowledge of Python is helpful. Some experience with Docker would be helpful but not required for the last quarter of the tutorial.